tests/test_hdfs.py

```diff
@@ -64,21 +67,27 @@ def hdfs_with_kerberos(configure_security_hdfs):
                 )
             )
         principals.append(GENERIC_HDFS_USER_PRINCIPAL)
+        principals.append(ALICE_PRINCIPAL)
```

Comment:
See https://github.com/mesosphere/dcos-commons/blob/master/frameworks/hdfs/tests/test_kerberos_auth.py#L47 for a more consistent way of handling Kerberos principals and setting up the kerberos environment.

Reply:
Sounds good. Let's address this in a follow-up ticket then.